KebPOS/Services/ProductService.cs

```diff
@@ -1,180 +1,284 @@
-﻿using KebPOS.Models;
+﻿using KebPOS.DbContexts;
+using KebPOS.Models;
+using Spectre.Console;
 
-namespace KebPOS.Services
+namespace KebPOS.Services;
+
+public class ProductService
 {
-    public class ProductService
+    internal static void InsertProduct()
     {
-        public static List<Product> GetProducts()
+        bool exit = false;
+        bool addNewProduct = true;
+        
+        while (addNewProduct)
         {
-            var products = new List<Product>()
+            bool isDuplicate = true;
```

Comment:
I think this method does the job but it ended up a bit verbose. I'll create a separate ticket to outsource the validation of the name into it's own method in the validation class with all the necessary checks. 🤔